locales/en.json

```diff
@@ -213,12 +213,20 @@
       "zero": "No message",
       "one": "You have 1 message",
       "other": "You have {{count}} messages"
-    }
+    },
+    "headerTitle": "io.italia.it",
```

Comment:
why is `io.italia.it` header title specific for messages? shouldn't it be the same for all screens? - @matteodesanti @fpersico 

@fgiobergia what are we displaying in other screens?